### PR TITLE
Parlai container

### PR DESCRIPTION
**Pull request by mojtaba-komeili**
# Patch description
The Dockerfile for building a working image for ParlAI.

## Note
A tutorial on using this Docker image will follow.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+RUN apt install -y git curl
+
+# Installing Anaconda
+WORKDIR /root
+RUN curl https://repo.anaconda.com/archive/Anaconda3-2021.05-Linux-x86_64.sh -o anaconda_installer.sh
+RUN bash anaconda_installer.sh -b -p
+ENV PATH="/root/anaconda3/bin:$PATH"
+
+# Installing recommmended pre-requirements
+RUN conda install pytorch torchvision torchaudio -c pytorch-lts -c nvidia
```
Comment (klshuster) [inline Dockerfile:17]: should we specify torch version? or just assume we work with highest torch number?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# Download the ParlAI Github repo
+RUN git clone https://github.com/facebookresearch/ParlAI.git ~/ParlAI
+
+# Running ParlAI install
+RUN cd ~/ParlAI && \
+    pip install -r requirements.txt && \
+    python setup.py develop
+
+CMD ["parlai", "party"]
```
Comment (klshuster) [inline Dockerfile:33]: 😍

**Review by klshuster — APPROVED** [review]
awesome, just small comment about torch versioning

File: Dockerfile (hunk 1 above)
Comment (mojtaba-komeili) [reply Dockerfile:17]: Yeah, good point. Should we set it to the same version as the install requirements?

**Pull request merged**